SEUNAGBEYE/React-QuizComponent
# #160909698 Create the QuizQuestion component

### What does this PR do?
 Create the Quiz Component

### Description of Task to be completed?
Create the Quiz Question Component

### How should this be manually tested?
- fetch this branch with `git fetch origin ft-create-quiz-questions-component-160909698:ft-create-quiz-questions-component-160909698`
- Checkout to the branch
- Run `npm install`
- Run `npm start`
- Then a browser should open showing a blank page

### Any background context you want to provide?
Na

### What are the relevant pivotal tracker stories?
[#160909698](https://www.pivotaltracker.com/n/projects/2200608/stories/160909698)

### Screenshots (if appropriate)
NA
